platform mode javaEE renamed to jakartaEE

alwin-joseph opened the pull request:
To fix https://github.com/eclipse-ee4j/jakartaee-tck/issues/83

Signed-off-by: alwin-joseph <alwin.joseph@oracle.com>

scottmarlow commented on install/jakartaee/bin/ts.jte:
```diff
@@ -1972,9 +1972,9 @@ ts_home=${ts.home}
 ## platform.mode is used by jpa tests to know whether to use
 ## the Java EE Programming Model for service and port access.
-## platform.mode=javaEE		Java EE Programming Model
+## platform.mode=jakartaEE		Java EE Programming Model
```
Should we also change ```Java EE``` to ```Jakarta EE``` for this pr?

alwin-joseph replied:
> @alwin-joseph
> I didn't finish reviewing yet but am thinking that further (separate) changes should be separate commits.
Yes,  there are more such references to Java EE, I suggest those can go in  another PR if possible.
> 
> Also, I'm concerned about merging the change to Standalone TCKs before teams working on changing Standalone TCKs are done with their changes (someone will have to resolve conflicts.) Should we merge our changes last? 
I'm not sure that helps because some of our changes we want to merge before we consider a standalone TCK complete for ballot.

I think there would be conflicts mostly in https://github.com/eclipse-ee4j/jakartaee-tck/pull/415. But I suggest merging this change first as it would be easier to resolve conflicts in other PRs. I will take care of resolving conflicts occurring in other documentation PRs due to this change (assuming I would have the permissions to do that in other changes)

scottmarlow commented:
@alwin-joseph 
I didn't finish reviewing yet but am thinking that further (separate) changes should be separate commits.

Also, I'm concerned about merging the change to Standalone TCKs before teams working on changing Standalone TCKs are done with their changes (someone will have to resolve conflicts.)  Should we merge our changes last?  I'm not sure that helps because some of our changes we want to merge before we consider a standalone TCK complete for ballot.

edbratt commented:
Are we proposing this just for the CTS (Platform TCK), or all TCKs form
JakartaEE TCK project? If the latter, this will require all these script
/ property files to be updated. If we only do this for the Platform TCK
and not the others, there will likely be confusion.

I don't oppose this change but doing this now, could set us back in
other TCKs.

My overall vote would be to defer this and do it once all TCKs and their
respective API projects have been completed for Jakarta 9 ballot.

On 8/8/2020 7:15 AM, Alwin Joseph wrote:
>
> *@alwin-joseph* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In install/jakartaee/bin/ts.jte
> <https://github.com/eclipse-ee4j/jakartaee-tck/pull/417#discussion_r467469103>:
>
> > @@ -1972,9 +1972,9 @@ ts_home=${ts.home}
>  #######################################################################
>  ## platform.mode is used by jpa tests to know whether to use
>  ## the Java EE Programming Model for service and port access.
> -## platform.mode=javaEE		Java EE Programming Model
> +## platform.mode=jakartaEE		Java EE Programming Model
>
>     @alwin-joseph <https://github.com/alwin-joseph>
>     I didn't finish reviewing yet but am thinking that further
>     (separate) changes should be separate commits.
>     Yes, there are more such references to Java EE, I suggest those
>     can go in another PR if possible.
>
>     Also, I'm concerned about merging the change to Standalone TCKs
>     before teams working on changing Standalone TCKs are done with
>     their changes (someone will have to resolve conflicts.) Should we
>     merge our changes last?
>     I'm not sure that helps because some of our changes we want to
>     merge before we consider a standalone TCK complete for ballot.
>
> I think there would be conflicts mostly in #415
> <https://github.com/eclipse-ee4j/jakartaee-tck/pull/415>. But I
> suggest merging this change first as it would be easier to resolve
> conflicts in other PRs. I will take care of resolving conflicts
> occurring in other documentation PRs due to this change (assuming I
> would have the permissions to do that in other changes)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eclipse-ee4j/jakartaee-tck/pull/417#discussion_r467469103>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC5WM3UJ3MTSN2QLV63WL73R7VMWNANCNFSM4PYUOFMA>.
>

alwin-joseph commented:
> Are we proposing this just for the CTS (Platform TCK), or all TCKs form JakartaEE TCK project? If the latter, this will require all these script / property files to be updated.

The changes made are spread over few standalone TCKs too. I have changed all the script/property files also where platform.mode is mentioned as javaEE and replaced it with jakartaEE. Only if other teams are using this same property in the in their configuration to run a tck, then they will have to make this edit too.  

> If we only do this for the Platform TCK and not the others, there will likely be confusion. I don't oppose this change but doing this now, could set us back in other TCKs. My overall vote would be to defer this and do it once all TCKs and their respective API projects have been completed for Jakarta 9 ballot.
> 

I verified all standalone TCK runs with this change. The only action we need to do is to regenerate the Tcks in staged directory. 

If there is a strong opinion against this change being made now, we can defer this.

scottmarlow commented:
@alwin-joseph I am concerned about updating TCKs while SPEC API teams are working on them.

alwin-joseph commented:
> @alwin-joseph I am concerned about updating TCKs while SPEC API teams are working on them.

If you are referring to the conflicts due to the UG changes we can either merge this PR first and deal with the conflicts in others or wait for the #415 (expected to have conflicts with this change) to be merged before merging this PR. Would that address your concern.

edbratt commented:
So long as any changes like this are tested -- I'm okay with it. We are 
behind schedule though so I would elect to hold off anything that we 
can. You're call though. If you think the time is now, go for it!

On 8/10/2020 7:37 AM, Alwin Joseph wrote:
>
>     @alwin-joseph <https://github.com/alwin-joseph> I am concerned
>     about updating TCKs while SPEC API teams are working on them.
>
> If you are referring to the conflicts due to the UG changes we can 
> either merge this PR first and deal with the conflicts in others or 
> wait for the #415 
> <https://github.com/eclipse-ee4j/jakartaee-tck/pull/415> (expected to 
> have conflicts with this change) to be merged before merging this PR. 
> Would that address your concern.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/eclipse-ee4j/jakartaee-tck/pull/417#issuecomment-671394872>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AC5WM3UQEARHU2R6RT6HP2TSAAA3TANCNFSM4PYUOFMA>.
>

alwin-joseph commented:
I have resolved the conflicts after the #415  merge and will be merging this PR now.
